magpy/gui/magpy_gui.py

```diff
@@ -2872,6 +2891,48 @@ def onChangeSymbol(self, event):
             self.OnPlot(self.plotstream,self.shownkeylist)
         self.changeStatusbar("Ready")
 
+    def OnFlagClick(self, event):
+        """Mouse event for flagging with double click."""
+        if not event.inaxes or not event.dblclick:
+            return
+        sensid = self.plotstream.header.get('SensorID','')
+        dataid = self.plotstream.header.get('DataID','')
+        if sensid == '' and not dataid == '':
+            sensid = dataid[:-5]
+        if sensid == '':
```

Comment:
Is the else for this `if` needed, or can you just `return` at the end of the if block?